packages/server/graphql/types/MeetingTemplate.ts

```diff
@@ -26,7 +27,12 @@ export const meetingTemplateFields = () => ({
     type: new GraphQLNonNull(GraphQLBoolean),
     description:
       'True if template is available to all teams including non-paying teams, else false',
-    resolve: ({isFree}: {isFree?: boolean}) => !!isFree
+    resolve: async ({id: templateId, isFree}: {id: string; isFree?: boolean}) => {
+      const endTimes = await db.readMany('endTimesByTemplateId', [templateId])
```

Comment:
- Looks like we are checking if the template was used by **any user** in parabol database and make it free. Should we only check if the viewer **organization** was using it? 

- Do we need to use dataLoader here? If we list templates, for every template we would do that query? If cache is missing we will do a [rethinkdb query](https://github.com/ParabolInc/parabol/blob/89228ca101fea2efcb38dcfadbf989f676d7e50e/packages/server/dataloader/customRedisQueries.ts#L13-L22).